dts/bindings/bluetooth/bt-hci.yaml

```diff
@@ -21,8 +21,8 @@ properties:
       - "i2c"
       - "smd"
       - "virtio"
-      - "ipm"    # Deprecated. "ipc" should be used instead.
       - "ipc"
+      - "ipm"    # Deprecated. "ipc" should be used instead.
```

Comment:
I would consider just removing `ipm` completely (also the define for it), so you don't have to play tricks with this exception.

Reply:
I am not aware of the deprecation policy so I avoid removing it. I will do it.